src/Adapters/Storage/Mongo/MongoStorageAdapter.js

```diff
@@ -81,15 +84,24 @@ export class MongoStorageAdapter {
     });
   }
 
-  // Used for testing only right now.
-  allCollections() {
-    return this.connect().then(() => {
-      return this.database.collections();
-    }).then(collections => {
+  // Delete all data known to this adatper. Used for testing.
+  deleteAllSchemas() {
+    return this._allCollections().then(collections => {
+      let promises = collections.map(collection => collection.drop());
+      return Promise.all(promises);
+    });
+  }
+
+  _allCollections() {
+    return this.connect()
+    .then(() => this.database.collections())
+    .then(collections => {
       return collections.filter(collection => {
         if (collection.namespace.match(/\.system\./)) {
           return false;
         }
+        //TODO: If you have one app with a collection prefix that happens to be a prefix of another
```

Comment:
I don't think so. Imagine two app connecting to the DB, one with collectionPrefix = `abc` and the other with collectionPrefix = `abcdef` and you deleted all collections that have `abc` as the prefix, you will get all the collections from both apps.